test/unit_tests/test_metadata_table.py

```diff
@@ -85,8 +85,7 @@ def test_bad_type_name(self):
 
         #Exercise
         with self.assertRaises(Exception) as context:
-            tables = parse_metadata_file(filename, known_ddts,
-                                         self._DUMMY_RUN_ENV)
+            parse_metadata_file(filename, known_ddts, self._DUMMY_RUN_ENV)
```

Comment:
What is the point of all these whitespace changes?

Reply:
Well, now the entire call fits in one line below 80 characters and thus easily on a terminal screen. I'll change it back ...